CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,36 @@
+## Code of Conduct
+
+This Code of Conduct is adapted from [Rust's wonderful
+CoC](http://www.rust-lang.org/conduct.html).
+
+* We are committed to providing a friendly, safe and welcoming
+  environment for all, regardless of gender, sexual orientation,
+  disability, ethnicity, religion, or similar personal characteristic.
+* Please avoid using overtly sexual nicknames or other nicknames that
+  might detract from a friendly, safe and welcoming environment for
+  all.
```

Comment:
@jasnell was just referring to [this](https://en.wikipedia.org/wiki/Widows_and_orphans).